src/index.html

```diff
@@ -0,0 +1,138 @@
+    <title>Google</title>
+  </head>
+  <body class="text-gray-800 text-sm font-Arial">
+    <header class="md:flex md:justify-end w-full">
+      <div class="hidden md:flex space-x-4 p-2 items-center justify-start">
+        <a href="#" class="hover:underline">Gmail</a>
+        <a href="#" class="hover:underline">Images</a>
+        <a href="#" class="hover:bg-gray-200 rounded-full p-4">
+          <img src="../dist/image/dots-menu.png" alt="" />
+        </a>
+        <a href="#" class="hover:bg-gray-200 rounded-full p-2">
+          <img
+            src="../dist/image/account_circle.svg"
+            alt="user"
+            class="w-8 h-8"
+          />
+        </a>
+      </div>
+      <div class="flex justify-between space-x-4 p-2 items-center md:hidden">
+        <div class="text-xs space-x-3 flex items-center">
+          <a href="#">
+            <svg
+              style="fill: #70757a; width: 24px; height: 24px"
+              viewBox="0 0 24 24"
+              xmlns="http://www.w3.org/2000/svg"
+            >
+              <path d="M0 0h24v24H0z" fill="none"></path>
+              <path d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"></path>
+            </svg>
+          </a>
+          <a href="#" class="text-[#4285f4] border-b-[#4285f4] border-b-2 p-2"
+            >ALL</a
+          >
+          <a
+            href="#"
+            class="p-2 hover:border-b-2 outline-none hover:border-b-black"
+            >IMAGES</a
+          >
+        </div>
+        <div class="flex">
+          <a href="#" class="hover:bg-gray-200 rounded-full p-4">
+            <img src="../dist/image/dots-menu.png" alt="" />
+          </a>
+          <a href="#" class="hover:bg-gray-200 rounded-full p-2">
+            <img
+              src="../dist/image/account_circle.svg"
+              alt="user"
+              class="w-8 h-8"
+            />
+          </a>
+        </div>
+      </div>
+    </header>
+    <main
+      class="container py-8 p-2 mx-auto max-w-md md:max-w-xl flex flex-col space-y-6"
+    >
+      <img
+        src="../dist/image/glue-google-color-logo.svg"
+        alt="goole"
+        class="h-24 mx-auto"
+      />
+      <div
+        class="rounded-full py-2 px-4 flex border border-gray-300 transition-all duration-500 hover:shadow-md"
+      >
+        <div class="place-self-center text-gray-400 cursor-pointer">
+          <i class="fas fa-search text-sm"></i>
+        </div>
+        <div class="flex px-4 w-full">
+          <input
+            type="text"
+            class="w-full border-none outline-none text-base"
+          />
+        </div>
+        <div class="place-self-center cursor-pointer">
+          <img src="../dist/image/google-voice.png" alt="" />
+        </div>
+        <div class="place-self-center cursor-pointer ml-4">
+          <img src="../dist/image/download.svg" alt="" class="w-8" />
+        </div>
+      </div>
+      <div class="mx-auto flex flex-row space-x-2 text-[#3c4043]">
```

Comment:
Instead of using arbitrary values, add this specific style value in the tailwind config file.